src/charts/raster-chart.js

```diff
@@ -353,10 +353,12 @@ export default function rasterChart(parent, useMap, chartGroup, _mapboxgl) {
       const layers = getLayers()
       const xRanges = []
       const yRanges = []
+      const polyRanges = []
```

Comment:
rename `viewRanges` or `viewBoxRanges` or similar

Reply:
Actually, it doesn't look like polyRanges is used anywhere. Is this a placeholder or can it be removed?

Reply:
@vastcharade it is removed now.